.circleci/config.yml

```diff
@@ -296,10 +352,38 @@ workflows:
               ignore:
                 - master
                 - py3-ci
+      - build-arm64:
+          filters:
+            branches:
+              ignore:
+                - master
+                - py3-ci
       - commit:
           requires:
             - build
           filters:
             branches:
               only:
                 - /.*ci-commit-master.*/
+  build-python3:
+    jobs:
+      - build-py-38:
+          filters:
+            branches:
+              only:
+                - py3-ci
+      - build-py-38-arm64:
+          filters:
+            branches:
+              only:
+                - py3-ci
```

Comment:
Same, lets run it for all branches except master like line 358-360

Reply:
Done!